plugins/plugin-vue/plugin.js

```diff
@@ -37,7 +38,23 @@ function displayError({contents, filePath, error}) {
   return output.join('\n');
 }
 
-module.exports = function plugin(snowpackConfig) {
+module.exports = function plugin(snowpackConfig, pluginOptions = {}) {
+  // Enable proper tree-shaking for Vue's ESM bundler
+  // See http://link.vuejs.org/feature-flags
+  const packageOptions = snowpackConfig.packageOptions || snowpackConfig.installOptions;
+  if (packageOptions && packageOptions.source === 'local') {
```

Comment:
@natemoo-re made a small change:

```diff
- if (packageOptions.source === 'local') {
+ if (packageOptions && packageOptions.source === 'local') {
```

The `plugin-vue` test which doesn’t fully load Snowpack was throwing because the default wasn’t set. While that’s probably more the test’s concern than the plugin, I still think it’s generally good to keep plugin test writing _not_ loading the entire Snowpack runtime, hence the change here.

I wasn’t really sure what gets normalized IRT the Snowpack config, so I just played it safe here. But if you‘d prefer throwing an error if `packageOptions` doesn’t exist instead, I could do that, too.